flow/Platform.actor.cpp

```diff
@@ -4056,7 +4084,7 @@ TEST_CASE("/flow/Platform/getMemoryInfo") {
 	                         "AnonHugePages:   1275904 kB\n";
 
 	std::stringstream memInfoStream1(memString1);
-	getMemoryInfo(request, memInfoStream1);
+	linux_os::getMemoryInfo(request, memInfoStream1);
```

Comment:
This seems to be guarded by `#if defined(__linux__)`?

Reply:
Ditto to above, the whole test is inside a `#ifdef __linux__` `#endif` block.